odoo/cli/populate.py

```diff
@@ -22,38 +22,68 @@ def run(self, cmdargs):
         group.add_option("--models",
                          dest='populate_models',
                          help="Comma separated list of model or pattern (fnmatch)")
+        group.add_option("--profile",
+                         dest='profiling_enabled', action="store_true",
+                         help="Specify if you want to profile records population.",
+                         default=False)
+        group.add_option("--rollback",
+                         dest='populate_rollback', action="store_true",
+                         help="Specify if you want to rollback database population.",
+                         default=False)
         parser.add_option_group(group)
         opt = odoo.tools.config.parse_config(cmdargs)
         populate_models = opt.populate_models and set(opt.populate_models.split(','))
         population_size = opt.population_size
+        profiling_enabled = opt.profiling_enabled
         dbname = odoo.tools.config['db_name']
         registry = odoo.registry(dbname)
         with registry.cursor() as cr:
             env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
-            self.populate(env, population_size, populate_models)
-
+            self.populate(
+                env, population_size, populate_models,
+                profiling_enabled=profiling_enabled,
+                rollback=opt.populate_rollback)
```

Comment:
I wonder if the rollback shouldn't be manage here and pass a parameters `commit=not rollback` instead. Since the cursor is only used for the populate, we can rollback the connection, we don't need to create a savepoint.

We could maybe patch the cr.commit so that it will do nothing (or raise). This will avoid the context key and maybe have a safer rollback option in case any other populate logic tries to commit.